src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -923,6 +923,9 @@ def load_arguments(self, _):
         c.argument('description', help='the description of the gallery')
     with self.argument_context('sig update') as c:
         c.ignore('gallery')
+        c.argument('permissions', arg_type=get_enum_type(['Private', 'Groups']), arg_group='Sharing Profile',
```

Comment:
What does `Groups` mean?

Reply:
Type of gallery permission

Reply:
Could we get enum values from SDK definition?